src/main/webapp/app/modules/managerSurveysDashboard/managerSurveysDashboard.tsx

```diff
@@ -99,22 +99,19 @@ const ManagerSurveysDashboard = () => {
             </Dropdown.Item>
           </Dropdown.Menu>
         </Dropdown>
-        <Link to={surveyPath}>
-          <div className={'name-row'}>
-            <p>{survey.name}</p>
-          </div>
-        </Link>
-        {/* Render other survey details as needed */}
       </div>
     );
   }
 
   return (
-    <div>
-      <h1>Survey Dashboard</h1>
-      <ActiveSurveysSection />
-      <NotPublishedSurveysSection />
-      <ExpiredSurveysSection />
+    <div className={'manager-surveys-dashboard'}>
```

Comment:
It is unnecessary and imo looks bad, because the title `Manager Survey Dashboard` is smaller than sections names. My suggestion is to think of better styling or just add a class to the <h1> which will make the font bigger and bold or sth. It would be enough imo.